tools/build_defs/framework.bzl

```diff
@@ -365,7 +382,7 @@ def cc_external_rule_impl(ctx, attrs):
             empty.file,
             wrapped_outputs.log_file,
         ],
-        tools = depset([wrapped_outputs.script_file] + ctx.files.data),
+        tools = depset([wrapped_outputs.script_file], transitive = [data[DefaultInfo].default_runfiles.files for data in data_dependencies]),
```

Comment:
Just checking - does `default_runfiles.files` include the data files themselves?

It'd be good to have a test that actually exercises the runfiles from data as I think the existing test will only check the data files itself and only at runtime. I haven't checked but I don't think the current test for `data` uses the files at buildtime? @UebelAndre?

Reply:
As far as I can tell, it only does for _rules_ - if you have a dependency on a _file_ label, it isn't included.

This surfaced when I wrote that test you asked for :)